packages/commons/test/Deque.AxeCore.Commons.Test.csproj

```diff
@@ -31,4 +32,8 @@
   <PropertyGroup>
     <VSTestResultsDirectory>TestResults/$(TargetFramework)</VSTestResultsDirectory>
 	</PropertyGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Resources/sampleFile.txt" />
```

Comment:
I don't think it's necessary to embed this resource - the test code never reads it as an embedded resource, only as the source file.